distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -93,6 +93,11 @@ log4j2.logger.org_ops4j_pax_web.name = org.ops4j.pax.web
 log4j2.logger.org_ops4j_pax_web.level = WARN
 log4j2.logger.org_ops4j_pax_web_jsp.name = org.ops4j.pax.web.jsp
 log4j2.logger.org_ops4j_pax_web_jsp.level = WARN
+# Logging for the UnregisterWebAppVisitorWC has been set to ERROR as paxweb logs stacktraces at WARN on startup. This may be
+# related to PAXWEB-1117. When Karaf upgrades paxweb, we need to see if these stacktrace log messages go away (see DDF-3321)
```

Comment:
Do we have a ticket for the paxweb update in the future? It'd be nice to include a mention of this ticket in there so it can be tracked

Reply:
Added a note to the next karaf upgrade ticket.